playbooks/rhv/inventory.example

```diff
@@ -0,0 +1,99 @@
+[all:vars]
```

Comment:
I needed to add here
ansible_ssh_private_key_file = /path/to/engine_id_rsa

for the passwordless connection.

Reply:
But I don't see you added it to the openshift vars example file. Only for the oVirt playbooks..